scrapers/BBWHighway.yml

```diff
@@ -0,0 +1,40 @@
+name: "BBWHighway"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - bbwhighway.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $h3: //div[@class="indvideo"]//div[@class="footer"]
+    scene:
+      Title:
+        selector: //div[@class="pagetitle"]//h1/text()
+      Details: //div[@class="videocontent"]/p
+      #Tags:
+      #  Name:
+      #    selector: //p[@class="tags clearfix"]/a
+      Performers:
+        Name: //span[@class="tour_update_models"]/a/text()
+      Image:
+        selector: //div[@class="mejs-layers"]//img/@src
+        postProcess:
+          - replace:
+              - regex: ^
+                with: https://bbwhighway.com
```

Comment:
get the image from the script
```yaml
      Image:
        selector: //script[contains(.,"poster=")]
        postProcess:
          - replace:
              - regex: .*poster="([^"]+)".*
                with: https://bbwhighway.com/$1
```
and remove `cdp`
It is not needed (and slows down scraping)